Homework5/task2.py

```diff
@@ -0,0 +1,16 @@
+# Задача 28: Напишите рекурсивную функцию sum(a, b), 
+# возвращающую сумму двух целых неотрицательных чисел. 
+# Из всех арифметических операций допускаются только +1 и -1. 
+# Также нельзя использовать циклы.
+# *Пример:*
+# 2 2
+#     4
+a = int(input("Please enter the number A: "))
+b = int(input("Please enter the number B: "))
+def SumNumbers(a, b):
+    if 0 == b:
```

Comment:
Обычно записывают не так.
```suggestion
    if b == 0:
```